<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/wiki_icon.png</normaloff>assets/wiki_icon.png</iconset>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: rgb(18, 18, 18);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(34, 34, 34, 255));}
</string>
   </property>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(238, 2, 73);
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="search_widget" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;

border-radius: 20px;</string>
    </property>
    <widget class="QLabel" name="search_icon_label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>11</y>
       <width>18</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>assets/search_icon.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_line_edit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>541</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title_icon">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/wiki_font.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_my_read">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgba(238, 2, 73, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>My Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_home">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgba(238, 2, 73, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_sign_out">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgba(238, 2, 73, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sign out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_read_list">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgba(238, 2, 73, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Read List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_fav">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgba(238, 2, 73, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Favourites</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
